querydsl-codegen/src/main/java/com/querydsl/codegen/CodegenModule.java

```diff
@@ -56,6 +58,7 @@ protected void configure() {
         bind(EmbeddableSerializer.class);
         bind(ProjectionSerializer.class);
         bind(SupertypeSerializer.class);
+        bind(Function.class, DefaultVariableNameFunction.class);
```

Comment:
Actually we do not even need to inject it. The variableNameFunction is read from the Configuration object and passed in the EntityTypeFactory:
```
this.typeFactory = new ExtendedTypeFactory(processingEnv, entityAnnotations, typeMappings, queryTypeFactory, conf.getVariableNameFunction());
```